Morphic.Client/QuickStrip/QuickStripWindow.xaml.cs

```diff
@@ -366,16 +366,43 @@ private void Zoom(object sender, QuickStripSegmentedButtonControl.ActionEventArg
             _ = session.Apply(SettingsManager.Keys.WindowsDisplayZoom, percentage);
         }
 
-        private void OnMagnify(object sender, QuickStripSegmentedButtonControl.ActionEventArgs e)
+        /// <summary>Original magnifier settings.</summary>
+        private Dictionary<Preferences.Key, object?> magnifyCapture;
+        
+        private async void OnMagnify(object sender, QuickStripSegmentedButtonControl.ActionEventArgs e)
         {
             Countly.RecordEvent("toggle-magnify");
             if (e.SelectedIndex == 0)
             {
-                _ = session.Apply(SettingsManager.Keys.WindowsMagnifierEnabled, true);
+                // Enable lens mode at 200%
+                Dictionary<Preferences.Key, object?> settings = new Dictionary<Preferences.Key, object?>
+                {
+                    {SettingsManager.Keys.WindowsMagnifierMode, 3},
+                    {SettingsManager.Keys.WindowsMagnifierMagnification, 200},
+                    {SettingsManager.Keys.WindowsMagnifierEnabled, true},
+                };
+
+                if (this.magnifyCapture == null)
+                {
+                    // capture the current settings
+                    var capture = await Task.WhenAll(settings.Select(
```

Comment:
Seems like `SettingsManager` could use a
````
async Task<Dictionary<Preferences.Key, object?>> Capture(IEnumerable<Preferences.Key> keys)
````
 method for symmetry with the `Apply` method.  Should work like the single-item `Capture` currently in there except loop the `session.Keys.Add` line and return `preferences.GetValuesByKey()`.

What do you think?

Reply:
Yes, the thought had crossed my mind.

Done.